ws-nextjs-app/pages/[service]/send/[id]/SuccessScreen/index.tsx

```diff
@@ -10,14 +10,17 @@ import TickSvg from './svgs';
 import { useFormContext } from '../FormContext';
 import fallbackTranslations from '../fallbackTranslations';
 
-const DEFAULT_RETENTION_POLICY_DAY = '270';
-const DEFAULT_EMAIL = 'CannotFindEmail@bbc.co.uk';
-
 type Props = {
   title: string;
+  replyEmailAddress: string;
+  retentionPeriod: string;
 };
 
-const SuccessScreen = ({ title }: Props) => {
+const SuccessScreen = ({
+  title,
+  replyEmailAddress,
```

Comment:
Heya, would it not be better to put the replyEmailAddress and title in the translations config instead?

Reply:
I'm not too sure on this, since it comes through in the ugc-campaign-config so it can vary from campiagn to campaign, and it isn't specific to each service.